### ISSUE TITLE
New context menu option that can ask you to select the board before saving.

### ISSUE THREAD

**Issue by kirtan403** [chrschorn/oneclick-send-to-trello#2]
I would like to request a new feature. Adding a context menu option, similar to the current option that asks the user to save the selected thing to which board and list, rather than directly saving to settings all the time. 

There may be sometime reason to save to another board or another list. But because of settings it always saves to the same board and list. It would be great

**Comment by chrschorn** [comment]
Yeah this would be a cool feature. I imagine that the user could add new board/list combinations in the options menu, in some sort of list. Clicking the extension button would use the first one in the list. All the other combinations would only show up in the right-click menu. The combinations could be reordered in the options dialog with drag-and-drop.

I won't spend time on this for now, since I don't have the time and it's mostly outside of what this extension was intended for. Feel free to create a pull request though!

**Comment by webbrain-one** [comment]
Thanks for the request! I’ve opened PR #11 to add a context menu option that asks which board and list to save to before saving, instead of always using the default. This should address the behavior you’re looking for.
